extensions/wikidata/src/org/openrefine/wikidata/schema/entityvalues/ReconItemIdValue.java

```diff
@@ -33,6 +33,11 @@ public ReconItemIdValue(Recon recon, String cellValue) {
         super(recon, cellValue);
     }
 
+	@Override
+	public boolean isPlaceholder() {
+		return true;
```

Comment:
I would say we should only return `true` when the `ReconItemIdValue` is associated with a `Recon` object with a `New` judgment. If the entity exists, then this is not a placeholder.